### ISSUE TITLE
train WIDERFace dataset on other model, got MAP=0

### ISSUE THREAD

**Issue by Iranb** [open-mmlab/mmdetection#4084]
**Describe the bug**
E.g.  I want to train WIDERFace with retinanet,I modified configs/retinanet/retinanet_r50_fpn_1x_coco.py as 

```
_base_ = [
    '../../_base_/models/retinanet_r50_fpn.py',
    '../../_base_/datasets/wider_face.py',
    '../../_base_/schedules/schedule_1x.py', '../../_base_/default_runtime.py'
]

model = dict(bbox_head=dict(num_classes=1))

# optimizer
optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
total_epochs = 300
log_config = dict(interval=1)
checkpoint_config = dict(interval=5)

data = dict(
    samples_per_gpu=64,
    workers_per_gpu=2,
)
```

start train and i got :
```
2020-11-10 16:19:19,198 - mmdet - INFO - Epoch [1][42/51]       lr: 8.292e-04, eta: 14:01:12, time: 3.930, data_time: 3.048, memory: 7666, loss_cls: 0.9743, loss_bbox: 1.1459, loss: 2.1202
2020-11-10 16:19:21,614 - mmdet - INFO - Epoch [1][43/51]       lr: 8.492e-04, eta: 13:55:52, time: 2.416, data_time: 0.218, memory: 7666, loss_cls: 0.9077, loss_bbox: 1.1153, loss: 2.0230
2020-11-10 16:19:26,676 - mmdet - INFO - Epoch [1][44/51]       lr: 8.691e-04, eta: 14:06:04, time: 5.062, data_time: 1.902, memory: 7666, loss_cls: 1.0369, loss_bbox: 1.1176, loss: 2.1545
2020-11-10 16:19:27,802 - mmdet - INFO - Epoch [1][45/51]       lr: 8.891e-04, eta: 13:53:35, time: 1.126, data_time: 0.116, memory: 7666, loss_cls: 0.9812, loss_bbox: 1.1330, loss: 2.1142
2020-11-10 16:19:34,622 - mmdet - INFO - Epoch [1][46/51]       lr: 9.091e-04, eta: 14:13:05, time: 6.819, data_time: 0.174, memory: 7666, loss_cls: 1.0281, loss_bbox: 1.1142, loss: 2.1423
2020-11-10 16:19:35,816 - mmdet - INFO - Epoch [1][47/51]       lr: 9.291e-04, eta: 14:01:20, time: 1.194, data_time: 0.166, memory: 7666, loss_cls: 1.0332, loss_bbox: 1.1313, loss: 2.1645
2020-11-10 16:19:38,739 - mmdet - INFO - Epoch [1][48/51]       lr: 9.491e-04, eta: 13:59:14, time: 2.923, data_time: 0.200, memory: 7666, loss_cls: 1.0454, loss_bbox: 1.1909, loss: 2.2363
2020-11-10 16:19:39,783 - mmdet - INFO - Epoch [1][49/51]       lr: 9.690e-04, eta: 13:47:29, time: 1.044, data_time: 0.162, memory: 7666, loss_cls: 1.0382, loss_bbox: 1.1155, loss: 2.1538
2020-11-10 16:19:41,929 - mmdet - INFO - Epoch [1][50/51]       lr: 9.890e-04, eta: 13:41:46, time: 2.143, data_time: 0.590, memory: 7666, loss_cls: 1.0157, loss_bbox: 1.2611, loss: 2.2768
2020-11-10 16:19:44,834 - mmdet - INFO - Exp name: retinanet_r50_fpn_1x_coco.py
2020-11-10 16:19:44,834 - mmdet - INFO - Epoch [1][51/51]       lr: 1.009e-03, eta: 13:40:06, time: 2.909, data_time: 0.112, memory: 7666, loss_cls: 1.0052, loss_bbox: 1.3851, loss: 2.3904
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 3232/3226, 229.7 task/s, elapsed: 14s, ETA:     0s

2020-11-10 16:20:03,276 - mmdet - INFO - 
+-------+-------+--------+--------+-------+
| class | gts   | dets   | recall | ap    |
+-------+-------+--------+--------+-------+
| face  | 39707 | 215710 | 0.012  | 0.000 |
+-------+-------+--------+--------+-------+
| mAP   |       |        |        | 0.000 |
+-------+-------+--------+--------+-------+
2020-11-10 16:20:03,285 - mmdet - INFO - Exp name: retinanet_r50_fpn_1x_coco.py
2020-11-10 16:20:03,285 - mmdet - INFO - Epoch(val) [1][51]     mAP: 0.0000
```
**Reproduction**
1. What command or script did you run?
```
./tools/dist_train.sh configs/WIDERFace_model/retinanet/retinanet_r50_fpn_1x_coco.py 8  --work-dir=./workdir/retinanet

```
2. Did you make any modifications on the code or config?  yes
    Did you understand what you have modified? yes 
    the config file :

```
_base_ = [
    '../../_base_/models/retinanet_r50_fpn.py',
    '../../_base_/datasets/wider_face.py',
    '../../_base_/schedules/schedule_1x.py', '../../_base_/default_runtime.py'
]

model = dict(bbox_head=dict(num_classes=1))

# optimizer
optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
total_epochs = 300
log_config = dict(interval=1)
checkpoint_config = dict(interval=5)

data = dict(
    samples_per_gpu=64,
    workers_per_gpu=2,
)
```
3. What dataset did you use?
WIDERFace

**Environment**
sys.platform: linux
Python: 3.7.9 (default, Aug 31 2020, 12:42:55) [GCC 7.3.0]
CUDA available: True
GPU 0,1,2,3,4,5,6,7: GeForce RTX 2080 Ti
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 10.0, V10.0.130
GCC: gcc (Ubuntu 6.5.0-2ubuntu1) 6.5.0 20181026
PyTorch: 1.3.1
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - Intel(R) Math Kernel Library Version 2020.0.2 Product Build 20200624 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v0.20.5 (Git Hash 0125f28c61c1f822fd48570b4c1066f96fcb9b2e)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CUDA Runtime 10.0
  - NVCC architecture flags: -gencode;arch=compute_35,code=sm_35;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_50,code=compute_50
  - CuDNN 7.6.5
    - Built with CuDNN 7.6.4
  - Magma 2.5.0
  - Build settings: BLAS=MKL, BUILD_NAMEDTENSOR=OFF, BUILD_TYPE=Release, CXX_FLAGS=-fvisibility-inlines-hidden -std=c++11 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I/opt/conda/conda-bld/pytorch_1574381331675/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/include -fdebug-prefix-map=/opt/conda/conda-bld/pytorch_1574381331675/work=/usr/local/src/conda/pytorch-1.3.1 -fdebug-prefix-map=/opt/conda/conda-bld/pytorch_1574381331675/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place=/usr/local/src/conda-prefix -Wno-deprecated -fvisibility-inlines-hidden -fopenmp -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -O2 -fPIC -Wno-narrowing -Wall -Wextra -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Wno-stringop-overflow, DISABLE_NUMA=1, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=True, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_STATIC_DISPATCH=OFF, 

TorchVision: 0.4.2
OpenCV: 4.4.0
MMCV: 1.2.0
MMCV Compiler: GCC 6.5
MMCV CUDA Compiler: 10.0
MMDetection: 2.6.0+d3cf38d

**Comment by ZwwWayne** [comment]
It seems that your batch size is very large, but the learning rate is not scaled accordingly.

**Comment by twmht** [comment]
@ZwwWayne 

I also have this issue when training widerface. I've tried bunch of network but always having very bad result.

my batch size is set to 6 and learning rate is set to 1e-3, these settings works for other dataset, but not work with widerface.

**Comment by hellock** [comment]
We have not tuned parameters on WIDERFace. It is community contributed.
